pages/index.vue

```diff
@@ -81,19 +86,34 @@ const playItem = (item: TrackModel, group: IDiscoverableGroup) => {
     <template v-else>
       <template v-for="group in discoverGroups" :key="group.header?.id || 0">
         <PageHeading v-if="group.header" :level="3">
-          <a
-            v-if="typeof group.header.link === 'string'"
-            :href="group.header.link"
-          >
-            {{ group.header.title }}
-          </a>
-          <span v-else>{{ group.header.title }}</span>
+          <div class="flex items-center justify-between">
+            <div>
+              <a
+                v-if="typeof group.header.link === 'string'"
+                :href="group.header.link"
+              >
+                {{ group.header.title }}
+              </a>
+              <span v-else>{{ group.header.title }}</span>
+            </div>
+            <NuxtLink v-if="group.header.link" :to="group.header.link">
+              <ButtonStyled intent="secondary" size="small">
+                <span
+                  class="overflow-hidden text-ellipsis whitespace-nowrap text-center"
+                  >{{ t("track.a11y.see-all") }}</span
```

Comment:
Please take this text separate from the `a11y` context. As this text is visible, its not an accessibility-related reason we're having this translated.

Btw... I doubt many translators do have an idea what `a11y` means, so if you'd rather be about to find a new keyword to put those translations to, go ahead.

Reply:
In which context should the translation have to be added?

Reply:
I would add it to a `home.list.see-all` or something... The button is IMO not really related to tracks... otherwise I have another ticket where I'm taking up some things about translation, where I could pick this up later 👍